Dockerfile

```diff
@@ -46,6 +51,12 @@ RUN apk update \
 COPY --from=dist /out/restic /usr/local/bin/
 COPY --from=dist /out/LICENSE /usr/share/licenses/restic/
 
+# Install rclone.
+COPY --from=dist /go/bin/rclone.v1 /usr/local/bin/
```

Comment:
@alnikyur this layer may be combined with the same above. No need to produce another layer when it can be omitted.